packages/admin-next/dashboard/src/routes/products/product-create/utils.ts

```diff
@@ -56,10 +56,19 @@ export const normalizeVariants = (
     manage_inventory: variant.manage_inventory || undefined,
     allow_backorder: variant.allow_backorder || undefined,
     // TODO: inventory - should be added to the workflow
-    prices: Object.entries(variant.prices || {}).map(([key, value]: any) => ({
-      currency_code: key,
-      amount: getDbAmount(castNumber(value), key),
-    })),
+    prices: Object.entries(variant.prices || {})
+      .map(([key, value]: any) => {
+        if (key.startsWith("reg_")) {
+          // TODO: route needs to accept region prices as well
+          return undefined
+        } else {
```

Comment:
@fPolic - is someone working on fixing this?

Reply:
Not ATM as far as I know